src/formio/templates/editGridRow.ejs

```diff
@@ -1,14 +1,14 @@
 <dl class="utrecht-data-list utrecht-data-list--html-dl utrecht-data-list--rows">
-  {% for (const key in ctx.flattenedComponents) { %}
-    {% if (ctx.isVisibleInRow(ctx.flattenedComponents[key]) && ctx.flattenedComponents[key].label) { %}
```

Comment:
You removed the check that a label exists, right? Won't this change the behaviour of this if statement, or does `isVisibleInRow(component)` automatically mean it has a label as well?

Reply:
Yeah, because label is one of the required properties, and if it's missing, it'll just end up with an empty string but shouldn't break the layout. `key`, `type` and `label` are properties for which we enforce in our https://github.com/open-formulieren/formio-builder that a value must be provided/set, and this is also expressed in our Typescript types. So, we can assume that there will always be a label value in each component, even if it's not relevant (like for content component).

The `ctx.isVisibleInRow` is internal Formio stuff related to the logic. We don't expose all this logic to end-users (and we have a long-standing backend issue related to this), but in short - logic on/in repeating group items is not supported.

This template code is actually closer to the default template code of Formio which also loops over `ctx.eachComponent` rather than `ctx.flattenedComponents`, as the former skips over layout components while the latter doesn't.